src/main/java/com/inventorysystem/inventory/VendorProfitController.java

```diff
@@ -49,10 +50,16 @@ public String addProduct(Product product){
         return "main";
     }
 
-     /** GetMapping for /product endpoint
+     /** GetMapping for /view-products endpoint
      * @return all products entered.
      */
-    @GetMapping("/product")
+
+     @RequestMapping("/view-inventory")
```

Comment:
Made page name more descriptive and made sure that it mapped to the appropriate page

Reply:
Going from GetMapping to RequestMapping makes this method more ambiguous (handle all HTTP request, not just GET).  Any reason for this?